src/Components/AdminTools/AddNewCustomer.tsx

```diff
@@ -34,18 +41,23 @@ const AddNewCustomer: React.FC<CustomerAdminDataProps> = ({ customerData, editMo
     }
   }, [editMode, customerData])
 
-  const handleSave = () => {
+  const handleSave = async () => {
     const payload = {
       Name: name,
```

Comment:
these payload keys have been updated on backend , please check with pranav on this and make changes are required

Reply:
Done, I have resolved it